OpenAL32/alSource.cpp

```diff
@@ -2723,6 +2729,9 @@ AL_API ALvoid AL_APIENTRY alSourcePlayv(ALsizei n, const ALuint *sources)
     auto start_source = [&context,device](ALuint sid) -> void
     {
         ALsource *source{LookupSource(context.get(), sid)};
+        if(!source)
+            return;
```

Comment:
Same here. It was checked earlier under the same lock, and if it gets a null here there's bigger problems.